src/guiapi/include/MItem_tools.hpp

```diff
@@ -90,7 +90,15 @@ class MI_TEST_HEAT : public WI_LABEL_t {
 
 public:
     MI_TEST_HEAT();
+protected:
+    virtual void click(IWindowMenu &window_menu) override;
+};
 
+class MI_TEST_FANS_fine : public WI_LABEL_t {
+    static constexpr const char *const label = N_("Test FANs fine");
```

Comment:
the EN wording of this message is really strange, what is the intention? Does it mean, the test of the fans went ok?

Reply:
this is fan test with 24 measured points (normal test has 5 points). This is only for Zoltan and we will remove this menuitem in RC.